### ISSUE TITLE
Failed to parse marco syntax when containing brackets

### ISSUE THREAD

**Issue by unlimitedsola** [asciidoctor/asciidoctor-intellij-plugin#411]
AsciiDoc Plugin Version: 0.30.38

IDE Name and Version: IntelliJ IDEA 2019.3.3 (Ultimate Edition)

In the following example, the brackets at the end of the line are parsed and highlighted wrongly.
After performing the Reformat Code action, the brackets get removed and broke my document unexpectedly.

```
= My Document
Sola <dev@sola.love>
:y: icon:check[role="green"]
:n: icon:times[role="red"]
```

Please let me know if additional information is needed.

**Comment by ahus1** [comment]
Sorry to hear that there is a regression in the latest release. The lexer fails parsing closing brackets in attributes. This will be fixed in the next release. I'll update this ticket once it is available.

**Comment by ahus1** [comment]
I've published version 0.30.39 that fixes this issue. It should be available from JetBrains plugin mirrors shortly.

It is also available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases).

**Comment by unlimitedsola** [comment]
Thanks for your quick fix, appreciate your hard work!
